docs/ssvc/SSVC.md

```diff
@@ -1,27 +1,35 @@
 # Stakeholder-Specific Vulnerability Categorization (SSVC)
 
+<!-- 
+note: the SSVC documentation site certcc.github.io/SSVC is slated to go online in mid-March 2024.
+Until then, it just redirects to an SSVC calculator https://certcc.github.io/SSVC/ssvc-calc/
+-->
+
 !!! abstract "Overview"
 
-    This section covers [Stakeholder-Specific Vulnerability Categorization](https://resources.sei.cmu.edu/library/asset-view.cfm?assetid=653459)
+    This section covers [Stakeholder-Specific Vulnerability Categorization](https://certcc.github.io/SSVC/)
     and how EPSS can be used with it.
 
     :technologist: [Source Code](https://github.com/epss-sig/epss-interoperability/blob/main/analysis/cisa_ssvc_dt/DT_analysis.ipynb) 
   
 
 ## SSVC
 
-[Stakeholder-Specific Vulnerability Categorization](https://resources.sei.cmu.edu/library/asset-view.cfm?assetid=653459) is proposed as an alternative to CVSS Base Scores.
+[Stakeholder-Specific Vulnerability Categorization](https://certcc.github.io/SSVC/) is proposed as an alternative to 
+CVSS Base Scores. SSVC provides a method for developing vulnerability management decision models that are tailored to the
+specific needs of different stakeholders. 
+
+It is based on research performed by the [CERT Division](https://www.sei.cmu.edu/about/divisions/cert/)
+of the [Software Engineering Institute](https://www.sei.cmu.edu) (SEI) at [Carnegie Mellon University](https://www.cmu.edu) (CMU):
+
+- [Prioritizing Vulnerability Response: A Stakeholder-Specific Vulnerability Categorization (Version 2.0)](https://resources.sei.cmu.edu/library/asset-view.cfm?assetid=653459)
+- [Github SSVC project](https://github.com/CERTCC/SSVC)
+- [SSVC Calculator](https://certcc.github.io/SSVC/ssvc-calc/)
+- [Stakeholder-Specific Vulnerability Categorization documentation](https://certcc.github.io/SSVC/)
 
-It is based on CMU SEI (Carnegie Mellon University Software
-Engineering Institute):
+### CISA and SSVC
```

Comment:
CISA's customized version of SSVC is an instance of SSVC, but not the whole thing. There should be a bit of narrative distance between "SSVC" as the thing CERT/CC is developing and CISA's implementation of SSVC for their own use. This change and the remaining changes are intended to highlight that distinction.